Gemfile

```diff
@@ -56,3 +58,9 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 group :production do
   gem 'pg', '0.18.4'
 end
+
+# replaces npm uglifier which does not work on heroku
+gem 'uglifier'
```

Comment:
What exactly do you mean by "does not work on heroku"? You should put an explanation in an extended commit message in 2b83a1e070ff34d68fb6a01bc86e71422c4283de.